runner/config/default.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
    * Service
    */
   serviceUrl: "http://localhost:3009", //This is used for redirects back to the runner.
-  serviceName: "Digital Form Builder - Runner",
+  serviceName: "Apply for funding",
```

Comment:
@RamuniN Should it be "Access funding"? Can you double check on the default naming please.